pkg/conversion/decode.go

```diff
@@ -100,7 +130,7 @@ func (s *Scheme) DecodeInto(data []byte, obj interface{}) error {
 // The function then implements the functionality of DecodeInto.
 // If specifiedVersion and specifiedKind are empty, the function degenerates to
 // DecodeInto.
-func (s *Scheme) DecodeIntoWithSpecifiedVersionKind(data []byte, obj interface{}, specifiedVersion, specifiedKind string) error {
+func (s *Scheme) DecodeIntoWithSpecifiedVersionKind(data []byte, obj interface{}, gvk unversioned.GroupVersionKind) error {
```

Comment:
Could you change the name of gvk to specifiedGVK and change the comment accordingly? Or leave a note in the comment saying the gvk contains the "specified" group/version/kind?